src/components/Messages.tsx

```diff
@@ -12,7 +12,7 @@ interface IMessages {
 
 export const Messages: React.FunctionComponent<IMessages> = ({ path, children }) => {
   const { messages, variant } = useDiagnostics(path);
-  const message = messages.map(m => m.summary || m.message).join(' | ');
+  const message = messages.map(m => m.message).join(' | ');
```

Comment:
yay! this is more sane. :)

Reply:
Urgh, I should have summary in place. It's still present on IDiagnostic.